README.md

```diff
@@ -1,2 +1,2 @@
 # Python_Programming_Learning
-This repository is for general learning purposes. Aim is to learn conventional way to write a python code.
+This is a fun game that asks players to guess the capita cities of diffrerent countries in the world.
```

Comment:
Some minor spelling corrections here. You can update, `capita` -> `capital` & `diffrerent` -> `different`.

Reply:
@ahernank sure ,thank you for pointing out the typo